data-es6.js

```diff
@@ -17291,6 +17291,44 @@ exports.tests = [
         hermes0_7_0: true,
         rhino1_7_13: false
       }
+    },
+    {
+      name: 'duplicate identifier',
+      exec: function(){/*
+        var d = function d([d]) { return d };
+        if (!d([true])) return false;
```

Comment:
```suggestion
        return !!(function d([d]) {});
```
Golf down the reproducing example a bit.

Reply:
I was actually checking that it returned the not-function binding, but I see I forgot the check explicitly for `!== true`.